src/ecosystem/Ecosystem.jl

```diff
@@ -45,3 +46,119 @@ function EnvironmentalLayer(
         ),
     )
 end
+
+"""
+    truncated_standard_normal_mean(a::Float64, b::Float64)::Float64
+
+Calculates the mean of the truncated standard normal distribution. Implementation taken
+from Distribution.jl excluding unused error checks
+
+# Arguements
+- 'a' : lower bound of the truncated distribution
+- 'b' : upper bound of the truncated distribution
+
+# Returns
+- Float64, mean of the truncated standard normal distribution
+
+"""
+function truncated_standard_normal_mean(a::Float64, b::Float64)::Float64
+    if abs(a) > abs(b)
+        return - truncated_standard_normal_mean(-b, -a)
+    elseif (a == b) 
+        return a
+    end
+    mid = (a + b) / 2 
+    Δ = (b - a) * mid
+    a′ = a * invsqrt2
```

Comment:
Where is `invsqrt2` and `sqrthalfπ` from? 

They look like variables that aren't defined?

Reply:
THey are precomputed constants from StatsFuns, I can replace them and remove the dependency?

Reply:
I suspected this, kind of surprised it's exported.

Options:

Either specify the package they're from for clarity:

```julia
a * StatsFun.invsqrt2
```

or add a note stating where they're from in the docstring (I'd be okay with this).

It's purely so future Dan/Takuya doesn't get confused.